src/tasks/__tests__/processBulkUpload.int.test.ts

```diff
@@ -350,6 +387,19 @@ describe('processBulkUpload', () => {
 			getMockJobHelpers(),
 		);
 		const updatedBulkUpload = await loadBulkUpload(bulkUpload.id);
+
+		const {
+			rows: [applicationForm],
+		} = await db.sql<ApplicationForm>('applicationForms.selectAll');
+		if (applicationForm === undefined) {
+			fail('The application form was not created');
+		}
+		expect(applicationForm).toMatchObject({
+			opportunityId: 1,
```

Comment:
@slifty Did you add a static reference here just to keep this review fun going?

Reply:
JKJKJK I think

Reply:
ugh (there are other places that reference opportunity ID directly too, and applicant ID.)

Reply:
(fixed both examples)